docs/architecture/microservices/multi-container-microservice-net-applications/subscribe-events.md

```diff
@@ -134,19 +134,19 @@ When implementing the steps of publishing the events, you have these choices:
 
 Figure 6-22 shows the architecture for the first of these approaches.
 
-![One approach to handle atomicity when publishing events: use one transaction to commit event to an event-log table, and then another transaction to publish (used in eShopOnContainers)](./media/image23.png)
+![Diagram of atomicity when publishing without a worker microservice.](./media/subscribe-events/atomicity-publish-event-bus.png)
 
 **Figure 6-22**. Atomicity when publishing events to the event bus
 
-The approach illustrated in Figure 6-22 is missing an additional worker microservice that is in charge of checking and confirming the success of the published integration events. In case of failure, that additional checker worker microservice can read events from the table and republish them, that is, repeat step number 2.
+One approach to handle atomicity when publishing events: use one transaction to commit event to an event-log table, and then another transaction to publish (used in eShopOnContainers). The approach illustrated in Figure 6-22 is missing an additional worker microservice that is in charge of checking and confirming the success of the published integration events. In case of failure, that additional checker worker microservice can read events from the table and republish them, that is, repeat step number 2.
```

Comment:
In this case we can leave this paragraph unchanged, as that's explained in the list ans paragraph above the figure.

```suggestion
The approach illustrated in Figure 6-22 is missing an additional worker microservice that is in charge of checking and confirming the success of the published integration events. In case of failure, that additional checker worker microservice can read events from the table and republish them, that is, repeat step number 2.
```